client/lib/plans/constants.js

```diff
@@ -42,10 +46,18 @@ export const FEATURE_NO_ADS = 'no-adverts';
 export const FEATURE_VIDEO_UPLOADS = 'video-upload';
 export const WORDADS_INSTANT = 'wordads-instant';
 
+// jetpack features constants
+export const FEATURE_DAILY_BACKUPS = 'jetpack-daily-backups';
+export const FEATURE_DAILY_MALWARE_SCANNING = 'jetpack-daily-malware-scanning';
+export const FEATURE_SPAM_PROTECTION = 'jetpack-spam-protection';
+export const FEATURE_REALTIME_BACKUPS = 'jetpack-realtime-backups';
+export const FEATURE_UNLIMITED_BACKUP_ARCHIVE = 'jetpack-unlimited-backup-archive';
+export const FEATURE_ONE_CLICK_THREAT_RESOLUTION = 'jetpack-one-click-threat-resolution';
+
 export const plansList = {
 	[ PLAN_FREE ]: {
 		getTitle: () => i18n.translate( 'Free' ),
-		getPriceTitle: () => i18n.translate( 'Free for life' ),
+		getPriceTitle: () => i18n.translate( 'Free for life' ), //TODO: DO NOT USE
```

Comment:
This is only used on the google analytics feature page, and contains currency information which will not be correct. For example we may have a user on an `en` locale, but has a currency_code of `EUR`, or `GBP`, or `CAD` or `AUD`